conf/README.md

```diff
@@ -0,0 +1,17 @@
+# Jekyll Configuration
+
+This folder contains different configuration files for the Jekyll build, which is triggered via `gulp build` and similar commands (`gulp serve` etc.).
+
+Files present after checkout:
+
+- `_config.yml` is the main Jekyll config file
+- `_version.yml` defines the latest version of the docs
+- `_dev.yml` is used for normal builds
```

Comment:
It might be better to say it's used for build by *default*, because "normal" builds are actually production builds, not dev builds. Dev builds are special.

Also, `_dev.yml` and `_prod.yml` are mutually exclusive: when `--prod` is specified, `_dev.yml` is not included.